x-pack/plugins/file_upload/server/routes.ts

```diff
@@ -314,6 +315,51 @@ export function fileUploadRoutes(coreSetup: CoreSetup<StartDeps, unknown>, logge
           const esClient = (await context.core).elasticsearch.client;
           const resp = await previewIndexTimeRange(esClient, timeField, pipeline, docs);
 
+          return response.ok({
+            body: resp,
+          });
+        } catch (e) {
+          return response.customError(wrapError(e));
+        }
+      }
+    );
+
+  /**
+   * @apiGroup FileDataVisualizer
+   *
+   * @api {post} /internal/file_upload/preview_tika_contents Returns the contents of a file using the attachment ingest processor
```

Comment:
Is it practical to add an API test for this new endpoint?

Reply:
we could yeah, we'd need a very small pdf file added to the test assets

Reply:
I've added it to our 8.16 test coverage issue - https://github.com/elastic/kibana/issues/188459.

Reply:
Thinking about this again, for the API test we'd need a base64 string of the file, so this could just be a string in the test file rather than an file that needs to be added to the test assets.